### DISCUSSION TITLE
Is CEBRA invariant to permutations in the input channels?

### DISCUSSION THREAD

**Discussion by icarosadero** [AdaptiveMotorControlLab/CEBRA#182] [Q&A] [answered]
Should I still get the same embedding even if I shuffle the order of the input neurons?

**Comment by stes** [comment, marked as answer]
Hi @icarosadero , you would not get the *same* embedding, but an embedding with very high consistency. So yes, the ordering does not matter.

This applies if you train and embed from scratch. The input layer is not permutation invariant in the sense that you can train a model, input permuted channels, and expect to get the same embedding.

Does that answer the Q?

**Reply by icarosadero** [reply]
Yes, it does. Thanks
